backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5703180_sys_added_OrderProcessing_Shopware_param.sql

```diff
@@ -0,0 +1,247 @@
+;
+
+-- Reference: Order Processing
+-- Value: N
+-- ValueName: Keine
+-- 2023-09-18T10:55:46.084838500Z
+UPDATE AD_Ref_List SET Value='N',Updated=TO_TIMESTAMP('2023-09-18 13:55:46.084','YYYY-MM-DD HH24:MI:SS.US'),UpdatedBy=100 WHERE AD_Ref_List_ID=543551
+;
+
+-- Reference: Order Processing
+-- Value: O
+-- ValueName: Auftrag
+-- 2023-09-18T10:56:04.875918200Z
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Reference_ID,AD_Ref_List_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541831,543557,TO_TIMESTAMP('2023-09-18 13:56:04.744','YYYY-MM-DD HH24:MI:SS.US'),100,'D','Y','Auftrag',TO_TIMESTAMP('2023-09-18 13:56:04.744','YYYY-MM-DD HH24:MI:SS.US'),100,'O','Auftrag')
+;
+
+-- 2023-09-18T10:56:04.876968900Z
+INSERT INTO AD_Ref_List_Trl (AD_Language,AD_Ref_List_ID, Description,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Ref_List_ID, t.Description,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Ref_List t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Ref_List_ID=543557 AND NOT EXISTS (SELECT 1 FROM AD_Ref_List_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Ref_List_ID=t.AD_Ref_List_ID)
+;
+
+-- Reference: Order Processing
+-- Value: S
+-- ValueName: Auftrag und Lieferung
+-- 2023-09-18T10:56:24.030205900Z
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Reference_ID,AD_Ref_List_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541831,543558,TO_TIMESTAMP('2023-09-18 13:56:23.906','YYYY-MM-DD HH24:MI:SS.US'),100,'D','Y','Auftrag und Lieferung',TO_TIMESTAMP('2023-09-18 13:56:23.906','YYYY-MM-DD HH24:MI:SS.US'),100,'S','Auftrag und Lieferung')
+;
+
+-- 2023-09-18T10:56:24.031250600Z
+INSERT INTO AD_Ref_List_Trl (AD_Language,AD_Ref_List_ID, Description,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Ref_List_ID, t.Description,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Ref_List t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Ref_List_ID=543558 AND NOT EXISTS (SELECT 1 FROM AD_Ref_List_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Ref_List_ID=t.AD_Ref_List_ID)
+;
+
+-- Reference: Order Processing
+-- Value: I
+-- ValueName: Auftrag, Lieferung und Rechnun
+-- 2023-09-18T10:56:40.554252500Z
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Reference_ID,AD_Ref_List_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541831,543559,TO_TIMESTAMP('2023-09-18 13:56:40.433','YYYY-MM-DD HH24:MI:SS.US'),100,'D','Y','Auftrag, Lieferung und Rechnun',TO_TIMESTAMP('2023-09-18 13:56:40.433','YYYY-MM-DD HH24:MI:SS.US'),100,'I','Auftrag, Lieferung und Rechnun')
+;
+
```

Comment:
Small Typo here: ``Rechnun -> Rechnung`` . Pls fix here and in model and also pls in general set the isTranslated flags for `de_DE`, `de_CH` and `en_US`

Reply:
done, except for the isTranslated flag.